invoicepilot_app/views.py

```diff
@@ -27,16 +28,24 @@ def get_object(self):
         return self.request.user.profile
 
 class Login(generics.CreateAPIView):
```

Comment:
In your Login class, you have assigned two different serializer classes to the serializer_class attribute. Won't the second assignment (serializer_class = LoginSerializer) overwrite the first one? If you want to use both serializers, you should have separate attributes for them.

Reply:
I'm sorry but this is not in my code. I assigned the LoginSerializer and only once